### PR TITLE
Add a new Submit Node for UUTAH CHPC

### PR DESCRIPTION
**Pull request by Mansalu**
I would like to register a submit node for researchers at the University of Utah CHPC.

The hostname of this is `osg-submit.chpc.utah.edu`

### CODE REVIEW

File: topology/University of Utah/Utah-Submit-Node/Utah-Submit-Node.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        Primary:
+          ID:  40a181cc3b7a43593188c9607310d35cb079bda9
+          Name: Mitchell Steinman
+      Site Contact:
+        Primary:
+          ID:  ddfa10c8a8500e5c31818e7c1a063b3ce5ab6a8b
+          Name: CHPC OSG Support
+    Description: Submit Node for University of Utah CHPC
+    FQDN: osg-submit.chpc.utha.edu
+    ID: x
```
Comment (Mansalu) [inline topology/University of Utah/Utah-Submit-Node/Utah-Submit-Node.yaml:22]:
```suggestion
    ID: x
```

I'm not sure how to get an ID for this.

Comment (matyasselmeci) [reply topology/University of Utah/Utah-Submit-Node/Utah-Submit-Node.yaml:22]: You can get one by cloning the repo and running `bin/next_resource_id`.
```suggestion
    ID: 1071
```

**Comment by matyasselmeci** [comment]
Looks like you also need to make a `SITE.yml` file in your new `Utah-Submit-Node` directory. You should copy `template-SITE.yml` and fill in the values. Use 10278 for the site ID.

**Comment by Mansalu** [comment]
Thanks

File: topology/University of Utah/Utah-Submit-Node/Utah-Submit-Node.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+          Name: Mitchell Steinman
+      Site Contact:
+        Primary:
+          ID:  ddfa10c8a8500e5c31818e7c1a063b3ce5ab6a8b
+          Name: CHPC OSG Support
+    Description: Submit Node for University of Utah CHPC
+    FQDN: osg-submit.chpc.utah.edu
+    ID: 1071
+    Services:
+      SubmitNode:
```
Comment (matyasselmeci) [inline topology/University of Utah/Utah-Submit-Node/Utah-Submit-Node.yaml:24]:
```suggestion
      Submit Node:
```

**Pull request merged**